scripts/test/test_compatibility_unix.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+PYTHON_VERSION="$1"
+OS_NAME="$2"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+echo "=== Testing aaz-dev-tools compatibility with Python ${PYTHON_VERSION} on ${OS_NAME} ==="
+
+# Create virtual environment
+python -m venv test_env
+source test_env/bin/activate
+
+# Upgrade pip and install build tools
+python -m pip install --upgrade pip setuptools wheel
+
+# Install built azdev wheel
+WHEEL_FILE=$(find dist/ -name "azdev-*.whl" | head -n1)
+if [ -z "$WHEEL_FILE" ]; then
+  echo "Error: No azdev wheel found"
+  exit 1
+fi
+echo "Installing azdev wheel: $WHEEL_FILE"
+python -m pip install "$WHEEL_FILE"
+
+# Install aaz-dev-tools requirements from shared file
+echo "Installing aaz-dev-tools dependencies..."
+python -m pip install --only-binary=:all: -r "${SCRIPT_DIR}/requirements.txt"
+
+# Test azdev can be imported and basic functionality works
+echo "Testing azdev import and basic functionality..."
+python "${SCRIPT_DIR}/test_imports.py"
```

Comment:
There is already a unit test, we don't need to check the function of azdev: https://github.com/Azure/azure-cli-dev-tools/blob/5130a24020ca0a47055e914a58a4af67d9eb98ad/azure-pipelines-cli.yml#L76

Reply:
@bebound , I've removed the github action pipeline tests - instead - I will run the tests offline and report the results on the PR comments. 

Are you ok to consider this resolved?

Reply:
Tested via the following external tests:

https://github.com/RenSilvaAU/compat-test-azdev/actions/workflows/cross-python-build-and-test.yml
https://github.com/RenSilvaAU/compat-test-azdev/actions/workflows/same-python-build-and-test.yml